renovate.json

```diff
@@ -1,46 +1,87 @@
+
 {
- "extends": ["config:base"],
- "baseBranches": ["master"],
- "bumpVersion": null,
- "packageRules": [
-   {
-     "depTypeList": ["devDependencies"],
-     "updateTypes": ["minor"],
-     "automerge": false
-   }
- ],
- "prHourlyLimit": 5,
- "prConcurrentLimit": 5,
- "rangeStrategy": "bump",
- "renovateFork": true,
- "includeForks": true,
- "regexManagers": [
+  "extends":[
+    "config:base"
+  ],
+  "enabled": "true",
+  "baseBranches": ["master"],
+  "timezone": "Europe/Helsinki",
```

Comment:
Could we use UTC instead of Europe/Helsinki if we need to pick a timezone? This repository is used by people all around the world.

Reply:
good point. Thanks Michael, I will change it to UTC